src/resources/container.js

```diff
@@ -60,6 +61,62 @@ class ContainerResource {
     instantiateRenderEntity(options) {
         return null;
     }
+
+    /**
+     * Queries the list of available material variants.
+     *
+     * @returns {object} A map which maps a variant name as a string to the variant index
```

Comment:
should the public interface simply return the array of strings? Or is the mapping from string index somehow useful to the user?

Reply:
Yeah, I think it's actually pointless to use the index, so this could just as well return strings.